### DISCUSSION TITLE
Org or multi repo spanning discussions

### DISCUSSION THREAD

**Discussion by rogeralsing** [community/community#2852] [Discussions] [answered]
Having some project that spans multiple programming languages and have multiple repositories.
It would be great to have a discussion for all of that, together.

Either by having a discussion on an org level, or better yet, discussions spanning multiple repositories.

It's painful to coordinate users and ideas of our tools between e.g. C#, Go and Kotlin implementations.

**Comment by probonopd** [comment]
I second this. I would like to move exsiting Discussions from repository to project level.

**Reply by jschumacher** [reply]
Another vote from me on org level discussions. It makes sense for projects with a single repository to have the discussion on the repo level, but we would want to build a community with a single hub and not split the discussions across multiple repos.

**Comment by TinoDidriksen** [comment, marked as answer]
Duplicate of https://github.com/github/feedback/discussions/41

**Reply by becca** [reply]
Thank you @TinoDidriksen! 

We are tracking this request in https://github.com/github/feedback/discussions/41 since there's a great level of detail there and we're aware of the need for bringing Discussions to communities who actively work across numerous repositories within a single organization. 🙇
